docs/pages/partials.mdx

```diff
@@ -0,0 +1,431 @@
+---
+title: Index of Partial Files
+description: For tech writers to easily see what content is available for reuse
+layout: tocless-doc
+---
+
+This page indexes the partial files (includes) that are re-used throughout the docs. The include is within the expandable `<Details>` block under each header. Some partials are only referenced but not directly included, because they include local relative links that do not work from the directory path where this index lives.
+
+## cloud\/
+
+### cloud\/call-to-action.mdx
+
+<Details title="cloud/call-to-action.mdx">
+
+(!docs/pages/includes/cloud/call-to-action.mdx!)
+
+</Details>
+
+### cloud\/cloudmanagedadvisory.mdx
+
+<Details title="cloud/cloudmanagedadvisory.mdx">
+
+(!docs/pages/includes/cloud/cloudmanagedadvisory.mdx!)
```

Comment:
This one is hidden due to scope—should we add `forScopes` in `config.json` for this page?